### PR TITLE
add  agent  it with proxy

### PR DESCRIPTION
**Pull request by yu199195**

### CODE REVIEW

File: shardingsphere-test/shardingsphere-agent-test/shardingsphere-agent-test-plugins/src/test/assembly/bin/start.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+if [ $# == 2 ]; then
+    MAIN_CLASS=${MAIN_CLASS}" "$1" "$2
+    echo "The port is $1"
+    echo "The configuration path is $DEPLOY_DIR/$2"
+    CLASS_PATH=${DEPLOY_DIR}/$2:${CLASS_PATH}
+fi
+
+echo "The classpath is ${CLASS_PATH}"
+
+nohup java ${JAVA_OPTS} ${JAVA_MEM_OPTS} -javaagent:../shardingsphere-agent.jar  -classpath ${CLASS_PATH} ${MAIN_CLASS} >> ${STDOUT_FILE} 2>&1 &
```
Comment (terrymanu) [inline shardingsphere-test/shardingsphere-agent-test/shardingsphere-agent-test-plugins/src/test/assembly/bin/start.sh:69]: No remote debug port added, but the compose service contains it

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: .github/workflows/it.yml [hunk 2]
```diff
@@ -191,3 +191,24 @@ jobs:
...
+            ${{ runner.os }}-maven-
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Build Project
+        run: ./mvnw -B clean install -Dmaven.javadoc.skip=true -Drat.skip=true -Djacoco.skip=true -DskipITs
+      - name: Run Integration Test
+        run: ./mvnw -B clean install -f shardingsphere-test/shardingsphere-integration-agent-test/pom.xml -Pit.env.agent
```
Comment (dmsolr) [inline .github/workflows/it.yml:214]: What does the first building for?
Providing the building archive for IT? If so, you don't need the `clean` goal.

File: shardingsphere-test/shardingsphere-integration-agent-test/shardingsphere-integration-agent-test-plugins/src/test/assembly/shardingsphere-proxy-agent-assembly.xml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    <includeBaseDirectory>true</includeBaseDirectory>
+
+    <fileSets>
+        <fileSet>
+            <directory>${project.basedir}/../../../shardingsphere-agent/shardingsphere-agent-plugins/shardingsphere-agent-plugin-logging/target/plugins</directory>
+            <outputDirectory>./plugins</outputDirectory>
+            <includes>
+                <include>**.jar</include>
+            </includes>
+        </fileSet>
```
Comment (dmsolr) [inline shardingsphere-test/shardingsphere-integration-agent-test/shardingsphere-integration-agent-test-plugins/src/test/assembly/shardingsphere-proxy-agent-assembly.xml:33]: Why do not you copy directly the archive of  `shardingsphere-agent`?

File: .github/workflows/it.yml (hunk 2 above)
Comment (yu199195) [reply .github/workflows/it.yml:214]: mvn  clean  only scope shardingsphere-test/shardingsphere-integration-agent-test/pom.xml， thanks

**Review by terrymanu — APPROVED** [review]

**Pull request merged**